lib/awful/tag.lua

```diff
@@ -785,6 +866,10 @@ end
 
 --- Add a signal to all attached tags and all tags that will be attached in the
 -- future. When a tag is detached from the screen, its signal is removed.
+--
+-- Do not use.
```

Comment:
Huh? Perhaps some word about why? I don't see the reason (nor the replacement function)